sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveExplainSuite.scala

```diff
@@ -182,9 +182,9 @@ class HiveExplainSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
          |GlobalLimit 1
          |+- LocalLimit 1
          |   +- AnalysisBarrier
-         |         +- Aggregate [a#0], [a#0, count(1) AS count#0L]
-         |            +- Project [_1#0 AS a#0, _2#0 AS b#0]
-         |               +- LocalRelation [_1#0, _2#0]
+         |         +- Aggregate [a#0: int], [a#0: int, count(1) AS count#0L]
+         |            +- Project [_1#0: int AS a#0, _2#0: int AS b#0]
```

Comment:
Except the leaf nodes, I think we do not need to add the types for the other nodes. cc @cloud-fan

Reply:
thanks for the comment @gatorsmile. In some cases, the leaf nodes are not the first occurrence of an attribute (es. think to the Range generator): so in those cases the types would not be displayed. Is this fine?

Reply:
The input schema and the output schemas are what we really care. Our output of Explain is already pretty complex.